empyrical/tests/test_stats.py

```diff
@@ -989,6 +989,33 @@ def test_cagr_noisy(self, returns, add_noise):
             noisy_cagr_2,
             1)
 
+    def test_conditional_value_at_risk(self):
+        value_at_risk = self.empyrical.value_at_risk
+        conditional_value_at_risk = self.empyrical.conditional_value_at_risk
+
+        # A single-valued array will always just have a CVaR of its only value.
+        returns = np.random.normal(0, 0.02, 1)
+        expected_cvar = returns[0]
+        assert_almost_equal(
+            conditional_value_at_risk(returns, cutoff=0), expected_cvar,
+        )
+        assert_almost_equal(
+            conditional_value_at_risk(returns, cutoff=1), expected_cvar,
+        )
+
+        # Test a returns stream of 21 data points at different cutoffs.
+        returns = np.random.normal(0, 0.02, 21)
+
+        for cutoff in (0, 0.0499, 0.05, 0.20, 0.999, 1):
+            # Find the VaR based on our cutoff, then take the average of all
+            # values at or below the VaR.
+            var = value_at_risk(returns, cutoff)
```

Comment:
I had originally omitted a test as it felt like I was simply testing numpy's `percentile` function; but it's probably worth having one in case we ever change `value_at_risk`, so I'll add